internal/utils/build/utils.go

```diff
@@ -0,0 +1,36 @@
+package build
+
+import (
+	"fmt"
+
+	buildv1 "github.com/openshift/api/build/v1"
+	"github.com/rh-ecosystem-edge/kernel-module-management/internal/api"
+	"github.com/rh-ecosystem-edge/kernel-module-management/internal/constants"
+)
+
+const (
+	HashAnnotation = "kmm.node.kubernetes.io/last-hash"
+
```

Comment:
```suggestion
```

Reply:
The new line is here to separate the exported constant from the unexported one.